test/mocks/server/fatal_action_factory.cc

```diff
@@ -0,0 +1,12 @@
+#include "fatal_action_factory.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+MockFatalActionFactory::MockFatalActionFactory() = default;
+MockFatalActionFactory::~MockFatalActionFactory() = default;
```

Comment:
Please omit these. The compiler will define a default constructor and destructor on its own.